examples/emulator_examples/10_state_example.py

```diff
@@ -0,0 +1,55 @@
+from pybpodapi.protocol import Bpod, StateMachine
+my_bpod = Bpod(emulator_mode=True)
+sma = StateMachine(my_bpod)
+sma.add_state(
+    state_name='State1',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State2'},
+    output_actions=[])
+sma.add_state(
+    state_name='State2',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State3'},
+    output_actions=[])
+sma.add_state(
+    state_name='State3',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State4'},
+    output_actions=[])
+sma.add_state(
+    state_name='State4',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State5'},
+    output_actions=[])
+sma.add_state(
+    state_name='State5',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State6'},
+    output_actions=[])
+sma.add_state(
+    state_name='State6',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State7'},
+    output_actions=[])
+sma.add_state(
+    state_name='State7',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State8'},
+    output_actions=[])
+sma.add_state(
+    state_name='State8',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State9'},
+    output_actions=[])
+sma.add_state(
+    state_name='State9',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'State10'},
+    output_actions=[])
+sma.add_state(
+    state_name='State10',
+    state_timer=1,
+    state_change_conditions={Bpod.Events.Tup: 'exit'},
+    output_actions=[])
```

Comment:
This can be simplified to:
```suggestion
for i in range(9):
   sma.add_state(
       state_name='State{}'.format(i),
       state_timer=1,
       state_change_conditions={Bpod.Events.Tup: 'State{}'.format(i+1)},
       output_actions=[])
sma.add_state(
    state_name='State10',
    state_timer=1,
    state_change_conditions={Bpod.Events.Tup: 'exit'},
    output_actions=[])
```